releases/release-1.22/release-notes/maps/pr-102201-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 102201
+releasenote:
+  text: Graduate prefer nominated node to beta.
```

Comment:
There is a common way this graduations were phrased in past releases:
X, Y Z graduates to Beta and enabled by default <follow by a short explanation>